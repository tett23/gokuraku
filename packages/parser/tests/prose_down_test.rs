extern crate parser;

use insta::{assert_debug_snapshot, glob};
use std::fs;

#[test]
fn prose_down_test() {
    glob!("../../../fixtures/prose-down/*.pd", |path| {
        let input = fs::read_to_string(path).unwrap();
        let ast = parser::prose_down_parse(&input).unwrap();

        assert_debug_snapshot!(ast);
    });
}
