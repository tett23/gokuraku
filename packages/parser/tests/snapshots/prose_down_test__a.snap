---
source: packages/parser/tests/prose_down_test.rs
assertion_line: 12
expression: result
---
Ok(
    Document {
        blocks: [
            Paragraph(
            	ab
            ),
        ],
    },
)
